CoreScriptsRoot/Modules/PlayerDropDown.lua

```diff
@@ -215,8 +215,15 @@ local function GetBlockedPlayersAsync()
 			local request = HttpRbxApiService:GetAsync(apiPath)
 			blockList = request and game:GetService('HttpService'):JSONDecode(request)
 		end)
-		if blockList and blockList['success'] == true and blockList['userList'] then
-			return blockList['userList']
+		if success then
```

Comment:
is this necessary? it seems like "if blocklist" would take care of it, don't think we need to check success